numpy/core/tests/test_einsum.py

```diff
@@ -918,6 +920,13 @@ def test_path_type_input(self):
         opt = np.einsum(*path_test, optimize=exp_path)
         assert_almost_equal(noopt, opt)
 
+    def test_spaces(self):
+        #gh-10794
+        arr = np.array([[1]])
+        for sp in itertools.product(['', ' '], repeat=4):
+            # no error for any spacing
+            np.einsum('{}...a{}->{}...a{}'.format(sp), arr)
```

Comment:
Should either have `{0}` for all, or maybe `{sp}...'.format(sp=sp)`

Reply:
Meant to use a * here